takhojin/utils/checkToken.js

```diff
@@ -0,0 +1,18 @@
+const secreteKey = process.env.secretKey;
+const jwt = require("jsonwebtoken");
+
+const checkToken = async (req, res, next) => {
+  const token = req.headers.authorization;
+  try {
+    jwt.verify(token, secreteKey);
+    next();
+  } catch {
+    const err = new Error("INVALID_ACCESS_TOKEN");
+    console.log(err);
+    err.statuscode = 400;
+    res.status(400).json({ message: err.message });
```

Comment:
1. catch 문이 실행 되었다는 것은 이미 error가 발생 했기 때문에 새롭게 에러를 만들 필요가 없습니다.
2. 따라서 catch가 실행 되렀을 때 적절한 status code와 message을 반환해 주면 됩니다.
3. 해당 로직은 인증과 관련된 로직이기 때문에 status code는 400보다는 401이 더 적절할 것 같습니다.
```suggestion
    res.status(401).json({ message: "INVALID_ACCESS_TOKEN" });
```